_posts/2021-12-16-ngff.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: OME-NGFF 2021 updates
+tags: file-formats community
+categories: blog
+---
+
+In December 2020, we published two blog posts reporting on the development of
+next-generation file formats (NGFFs). We discussed public specifications for
+storing [multiscale images, label images]({{ site.baseurl }}{% post_url 2020-11-04-zarr-data %})
+and [high-content screening datasets]({{ site.baseurl }}{% post_url 2020-12-01-zarr-hcs %}).
+
+Here we report the progress made on NGFFs over the course of 2021 with contributions from OME, Glencoe Software, EMBL, Harvard Medical School.
+
+
+## NGFF paper
+
+The principle of next-generation file formats as a solution for bioimaging
+data storage and access was recently published as a
+[Brief Communication in Nature Methods](https://www.nature.com/articles/s41592-021-01326-w).
+This work was driven by OME and Glencoe Software, with significant
+contributions from our partners at EMBL and Harvard Medical School. The work
+reported here is driven by use cases in image data publication and domains
+with large bioimaging datasets-- public data resources, high content screening
+and light sheet microscopy. We have tried to make the work as widely
+applicable as possible, while trying to deliver examples of real world
+solutions.
+
+This paper includes a
+[benchmark](https://github.com/ome/bioimage-latency-benchmark) which measures
+the access times for different types of  data chunks. The reading performance
+of NGFFs is compared to the established TIFF and HDF5 format for different
+imaging modalities in different types of storage. These results demonstrate
+the benefits of the different formats under different access scenarios, e.g.,
+on a local computer, in the cloud, etc..  We discuss the advantages and
+tradeoffs of each format in various contexts.  We believe there is no single
+format that provides optimal performance in all scenarios. The corollary to
+this is that any imaging data-intensive project will have to consider these
+issues and make informed choices regarding the data structures they use. See
+the “[Outlook](https://www.nature.com/articles/s41592-021-01326-w#Sec4)”
+section of the paper for more details.
+
+## NGFF specifications
+
+In addition to the paper, several additions have been made to the
+[latest NGFF specification](https://ngff.openmicroscopy.org/latest/), currently at version 0.3.
+
+### Dimension separator
```

Comment:
This don't clearly render as sub-headings. I could see adding a prefix or adjusting the CSS.